README.md

```diff
@@ -134,7 +134,7 @@ import { createSignedUrl } from './auth_utils'
 app.get('/looker_auth', function(req, res) {
   // Authenticate the request is from a valid user here
   const src = req.query.src;
-  const host = 'http://looker.example.com:443'
+  const host = 'http://looker.example.com'
```

Comment:
This is my typo, but should this should probably be `https:`?

Reply:
Yup that makes sense to me